vuepress/docs/v7.1/docs/getting-started/ent-bundle.md

```diff
@@ -105,6 +105,8 @@ See the [Build and Publish a Simple Bundle](../../tutorials/create/pb/publish-si
 |`ent bundle pack`| Generate the bundle artifacts, the bundle image and the microservice images |
 |`ent bundle pack --org [organization]`| Generate the bundle artifacts and images, passing the organization name to Docker Hub |
 |`ent bundle pack --file [my-dockerfile]`| Use a custom Dockerfile for the bundle |
+|`ent bundle pack -s`| Create YAML descriptors and bundle the Dockerfile |
```

Comment:
is it bundling the Dockerfile?

Reply:
Nope. ent bundle pack --help says this "-s, --skip-docker-build  Skip the building of Docker images"

Reply:
sorry! meant to fix this; copied it before looking at the ENG ticket...